tasks/watch.js

```diff
@@ -3,7 +3,7 @@
 function setUpTaskWatch(tyRun) {
   var defaults = require('../lib/config/defaults');
 
-  var config = defaults.getInstance()
+  defaults.getInstance()
```

Comment:
Looking at the rest of this file, if there are no references to `config`, we can drop this statement. It was probably made redundant as part of the refactor making things more DRY.